packages/core/src/particle/modules/ParticleCompositeCurve.ts

```
@@ -175,6 +175,11 @@
         return this.constant;
       case ParticleCurveMode.TwoConstants:
         return this.constantMin + (this.constantMax - this.constantMin) * lerpFactor;
+      case ParticleCurveMode.Curve:
+        return this.curve?._evaluate(time);
+      case ParticleCurveMode.TwoCurves:
+        const min = this.curveMin?._evaluate(time);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)